src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.Sdk.FSharp.props

```diff
@@ -0,0 +1,24 @@
+<!--
+***********************************************************************************************
+Microsoft.NET.Sdk.FSharp.props
+
+WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and have
+          created a backup copy.  Incorrect changes to this file will make it
+          impossible to load or build your projects from the command-line or the IDE.
+
+Copyright (c) .NET Foundation. All rights reserved. 
+***********************************************************************************************
+-->
+
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+    <Choose>
+        <When Condition=" '$(MSBuildRuntimeType)' == 'core' ">
```

Comment:
I will chat with the sdk team tomorrow to find out about what happens with the sdk on mono.